extensions/ipynb/src/test/serializers.test.ts

```diff
@@ -107,6 +133,81 @@ suite('ipynb serializer', () => {
 				]
 			);
 		});
+		test('Multi-line Stream output', () => {
+			validateCellOutputTranslation(
+				[
+					{
+						name: 'stdout',
+						output_type: 'stream',
+						text: [
+							'Epoch 1/5\n',
```

Comment:
I think the tests would be more readable with simpler examples, that's just my personal preference though

Reply:
I thought of adding a more real world example, but will change to a simpler output.

Reply:
Done